components/ui/data-table/use-data-table-styles.tsx

```diff
@@ -18,7 +18,7 @@ export const useDataTableStyles = <TData, TValue>(input: useDataStylesInput<TDat
   const generateClassnameForHeader = (input: GenerateClassnameForHeaderInput) => {
     const { columnId } = input;
 
-    const baseClassnameForHeader = "border-b bg-white";
+    const baseClassnameForHeader = "border-b bg-white whitespace-nowrap";
```

Comment:
The reason I want this is because "Pediatric Age Group" is a long word and if you don't do this it will display on three lines in the header of the table and it looks really not good. This makes it all fit on one line and expands the header horizontally if it needs more space.
![image](https://github.com/serotracker/dashboard/assets/86806388/95779d48-cab5-4ca7-b8e2-08a5985c3e12)